### DISCUSSION TITLE
[BP#82] Optimiser la taille des cookies

### DISCUSSION THREAD

**Discussion by tbroyer** [cnumr/best-practices#230] [Bonnes pratiques] [answered]
https://groups.google.com/g/ecoconceptionweb/c/WyyzIK50DlQ/m/MpcOKnhyCQAJ

> “optimiser la taille des cookies”: est-ce qu'on pourrait une bonne fois pour toute arrêter de parler de "fichier" quand on parle de cookies ? Au passage, on peut mettre à jour la référence vers la RFC 6265, qui spécifie un fonctionnement proche de ce qui est fait par les navigateurs, quand les précédentes spécifications (2109 et 2965) tentaient de "standardiser" un "fonctionnement idéal" qui n'a jamais été adopté (Opera a supporté la RFC 2965 pendant un temps, mais aucun serveur n'a jamais envoyé de Set-Cookie2 en pratique, sauf expérimentation).

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé une issue pour améliorer le contenu de cette BP: https://github.com/cnumr/best-practices/issues/269
